std/string.d

```diff
@@ -4153,7 +4154,7 @@ unittest
         $(LREF entab)
   +/
 auto entabber(Range)(Range r, size_t tabSize = 8)
-    if (isForwardRange!Range)
+    if (isForwardRange!Range && isSomeChar!(ElementEncodingType!Range))
```

Comment:
No problem. I think you had a second part to your question.

> Is this additional constraint necessary?

I think the isSomeChar constraint is necessary as we are trying to change character.